### ISSUE TITLE
Cannot find module './provider-utils/undefined/index'

### ISSUE THREAD

**Issue by aireater** [aws-amplify/amplify-cli#918]
**Describe the bug**
Can't migrate a project to created with previous version of amplify cli. 

Error: 

```
✖ There was an error migrating your project.
migration operations are rolledback.
/Users/aireater/.nvm/versions/node/v11.2.0/lib/node_modules/@aws-amplify/cli/node_modules/gluegun/build/index.js:13
    throw up;
    ^

Error: Cannot find module './provider-utils/undefined/index'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:603:15)
    at Function.Module._load (internal/modules/cjs/loader.js:529:25)
    at Module.require (internal/modules/cjs/loader.js:658:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.keys.forEach (/Users/aireater/.nvm/versions/node/v11.2.0/lib/node_modules/@aws-amplify/cli/node_modules/amplify-category-api/index.js:17:38)
    at Array.forEach (<anonymous>)
    at Object.keys.forEach (/Users/aireater/.nvm/versions/node/v11.2.0/lib/node_modules/@aws-amplify/cli/node_modules/amplify-category-api/index.js:15:42)
    at Array.forEach (<anonymous>)
    at migrate (/Users/aireater/.nvm/versions/node/v11.2.0/lib/node_modules/@aws-amplify/cli/node_modules/amplify-category-api/index.js:13:28)
    at Array.categoryMigrationTasks.unshift (/Users/aireater/.nvm/versions/node/v11.2.0/lib/node_modules/@aws-amplify/cli/src/lib/migrate-project.js:103:44)

```

**To Reproduce**
1. Create a project with create react app using amplify cli version 0.1.36. 
2. add hosting
3. create a new AppSync API using the web console.
4. Add the api to the project using ```amplify add codegen --apiId [app id]```
5. upgrade to latest version of cli 1.1.3 
6. run ```amplify migrate``` to get error

**Expected behavior**
migrates project to new version of cli

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser N/A
 - Version 1.1.3

**Additional context**
I seem to get the same error for other amplify commands like status

**Comment by kaustavghosh06** [comment]
@aireater Fixed the issue in a PR #920. Will merge and release this by end of day today.

**Comment by kaustavghosh06** [comment]
We published a new version of the CLI with a fix for the same. Please try it out and re-open the issue if you're still seeing it.
